libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -132,6 +132,16 @@ def add_texts(
         # Insert the embedded texts in the vector store table.
         return self._insert_embeddings(texts, embedded_texts, metadatas, ids)
 
+    def drop(self) -> None:
+        """Drops every table created during initialization of vector store."""
+        logging.info(f"Dropping vector store: {self.table_name}")
```

Comment:
Can we add this log after successfully dropping?

Reply:
Yeah, we can add an additional log to indicate the end of the action.